deploy/cephfs/helm/templates/nodeplugin-clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "ceph-csi-cephfs.nodeplugin.fullname" . }}
+  labels:
+    app: {{ include "ceph-csi-cephfs.name" . }}
+    chart: {{ include "ceph-csi-cephfs.chart" . }}
+    component: {{ .Values.nodeplugin.name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list", "update"]
+  - apiGroups: [""]
+    resources: ["namespaces"]
+    verbs: ["get", "list"]
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
```

Comment:
Can you please double check the permssions for cephfs? rbd needs configmap reading, but I'm not sure cephfs does.

Reply:
taken this from https://github.com/ceph/ceph-csi/blob/csi-v1.0/deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml#L25-L27

Reply:
Ok. I'll talk with rootfs about it. But I don't think that should block this pr.